qt/python/mantidqt/mantidqt/widgets/superplot/presenter.py

```diff
@@ -237,6 +237,17 @@ def on_visibility_changed(self, visible):
                 pass
             self._canvas.draw_idle()
 
+    def on_normalise_checked(self, checked):
```

Comment:
Docstring for `checked` is missing here.

Reply:
Done